bin/read-config.ts

```diff
@@ -0,0 +1,17 @@
+import fs from 'fs';
+import yaml from 'js-yaml';
+import { template } from 'lodash';
+
+const regex = /\#[^*]*?$/gm;
+
+export default (path: string) => {
+  const data = fs.readFileSync(path, 'utf8');
+
+  // remove all comments
+  const configStr = data.replace(regex, '');
```

Comment:
don't do this. yaml is a very complicated format and this will cause edge case issues.
should let yaml parser does the remove comment work instead of just a regex
one way is parse to json and unparse back to yaml text, which shouldn't contain any comments.
but I don't see too much issue process `${}` inside comments.